sway/input/cursor.c

```diff
@@ -19,16 +20,12 @@ static void cursor_update_position(struct sway_cursor *cursor) {
 	cursor->y = y;
 }
 
-static void cursor_send_pointer_motion(struct sway_cursor *cursor,
-		uint32_t time) {
-	struct wlr_seat *seat = cursor->seat->wlr_seat;
-	struct wlr_surface *surface = NULL;
-	double sx, sy;
-
+static struct sway_container *cursor_at(struct sway_cursor *cursor,
```

Comment:
The function is `cursor_at`, but it doesn't return a cursor. Should be `surface_at_cursor_position()` or something similar.